final_task/rss_reader/classes/rss_feed.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+import jsonpickle
+
+from classes.logger import logger
+
+
+class RssFeed:
+    '''
+    Contatins RSS channel title, description, link and list of news
+    Contatins method for parsing class to json via jsonpickle module
+    '''
+    def __init__(self, title, description, link, news_list):
+        logger.log('INIT RSS FEED CLASS')
+        self.news_list = news_list
+        self.title = title  
+        self.description = description
+        self.link = link
+    
+    def print_feed(self):
+        result = '\n'
+        result += ' '*36 + self.title + '\n'
+        result += ' '*36 + '='*len(self.title) + '\n'
+        
+        #This line do some math to align descrtiption and title
+        result +=' '*int(abs((36 + len(self.title)/2 - len(self.description)/2))) + self.description + '\n\n'
+        result +='='*120 + '\n'
+        for _, item in enumerate(self.news_list):
+            result += str(item)  + '\n'
+            result += '='*120 + '\n'
+        print(result)
+    
+    def toJSON(self):
```

Comment:
You may want to check your code with `pycodestyle` tool.

Reply:
Done

Reply:
Also, function names should be in lowercase. preferrably using `snake_case_notation`